### PR TITLE
Multiplicity directory + multCalibrator first commit

### PR DESCRIPTION
**Pull request by ddobrigk**

### CODE REVIEW

**Comment by ddobrigk** [comment]
I cannot find the whitespace problem at all... I did run clang-format on all files, and I tried bash scripts for removing trailing whitespace, but there isn't any problem anywhere to be found. Am I missing something? Sorry if so.

**Comment by ddobrigk** [comment]
Turns out the reason for my confusion was that the trailing whitespaces the system complained about were not introduced by my changes - rather, they were already there before (I guess the checks changed with time). Should all be fine now!

File: CODEOWNERS [hunk 1]
```diff
@@ -28,3 +28,5 @@
 /PWGLF @alibuild @njacazio
 /PWGMM @alibuild @aalkin
 /Tools/PIDML @alibuild @saganatt
+/Common/Tools/Multiplicity @alibuild @ddobrigk @victor-gonzalez
```
Comment (jgrosseo) [inline CODEOWNERS:31]: Please move this up (behind Common)

File: Common/CMakeLists.txt [hunk 2]
```diff
@@ -15,3 +15,4 @@ add_subdirectory(CCDB)
 add_subdirectory(Tasks)
 add_subdirectory(TableProducer)
 add_subdirectory(Tools)
+add_subdirectory(Tools/Multiplicity)
```
Comment (jgrosseo) [inline Common/CMakeLists.txt:18]: Please add this to the CMakeLists.txt in Tools instead

Comment (ddobrigk) [reply Common/CMakeLists.txt]: Thanks, @jgrosseo, done! (was busy preparing LHCP talk)

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**